yunwoo/programmers-1/나누어-떨어지는-숫자-배열.js

```diff
@@ -0,0 +1,14 @@
+function solution(arr, divisor) {
+  const result = arr
+    .filter((value) => value % divisor === 0)
+    .sort((a, b) => a - b);
+
+  return result.length === 0 ? [-1] : result;
+}
```

Comment:
```suggestion
function solution(arr, divisor) {
  const result = arr
    .filter((value) => value % divisor === 0)
    .sort((a, b) => a - b);

  return !result.length ? [-1] : result;
}
```
- 와 filter 너무 깔끔하네요 너무 잘봤습니다 👍  도움이 되었습니다 🙆‍♂️
- 이건 코딩 스타일 차이인데 `result.length === 0` 도 좋지만 `!result.length` 도 괜찮을 것 같은 생각이 들었습니다 길이가 0이고 숫자 0은 비교 시에 false로 형변환이 되기 때문입니다

Reply:
- 제 코드가 민재님한테 도움이 되다니 이런날도 있군요😃
- not연산자를 리액트에서 쓸때는 이해가 잘됐는데 이럴땐 잘생각이 안나네요ㅠㅠ 참고하겠습니다 감사합니다 👍